extensions/ql-vscode/src/remote-queries/shared/remote-query-result.ts

```diff
@@ -1,6 +1,8 @@
 export interface RemoteQueryResult {
   queryTitle: string;
   queryFile: string;
```

Comment:
Now that we need the query name as well as the file path, I think `queryFile` isn't a good name any more. 

How about we make the following renames:
* queryFile -> queryName
* queryPath -> queryFilePath
* queryTextTmpFile -> queryTextTmpFilePath

Reply:
Done! Though I've used queryFileName instead of queryName (to avoid naming conflicts).

Reply:
That's a good call, I was actually conflating two different things.